.github/workflows/bench-turborepo.yml

```diff
@@ -40,3 +47,61 @@ jobs:
             -X POST \
             -H 'Authorization: Bearer ${{ secrets.TINYBIRD_TOKEN }}' \
             'https://api.us-east.tinybird.co/v0/datasources?format=ndjson&name=turbo_benchmarks&mode=append'
+
+  time-to-first-task:
+    name: Time to First Task
+    timeout-minutes: 60
+    runs-on: ${{ matrix.os.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: ubuntu
+            runner: ubuntu-latest
+          - name: macos
+            runner: macos-latest
+          - name: windows
+            runner: windows-latest
+
+    # Set the secret as an env var so we can use it in the script
+    env:
+      TINYBIRD_TOKEN: ${{secrets.TINYBIRD_TOKEN}}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-node
+      - uses: ./.github/actions/setup-go
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      # setup Rust on windows, because apparently setup-turborepo-environment doesn't do this
+      - name: Add rustup for windows
+        if: ${{ matrix.os.runner == 'windows-latest'}}
+        run: |
+          rustup target add aarch64-unknown-linux-musl
+          rustup target add x86_64-pc-windows-gnu
+          rustup target add x86_64-unknown-linux-musl
```

Comment:
Why do we need linux targets on windows?

Reply:
we don't, I didn't want to experiment with it though, the feedback cycle on these is 30-45 mins long. This is copy paste from the release job which sets up for windows and linux https://github.com/vercel/turbo/blob/e28a1c072cfc76b4cb28b3b0256d5ce66336f335/.github/workflows/turborepo-release.yml#L180-L183